models/src/main/protobuf/CasperMessage.proto

```diff
@@ -55,11 +56,19 @@ message Deploy {
   bytes sig   = 4; //signature of (hash(resource) + nonce) using private key
 }
 
+message DeployString {
```

Comment:
What's the difference between Deploy and DeployString?

Reply:
`Deploy` has the term already represented as a `Par` (the profobuf representing a Rholang term), whereas `DeployString` has the term represented as source code and so needs to be parsed to make a `Deploy`. Since the parse could fail I thought that it made sense to separate the two cases in order to isolate the logic handling the term itself from the parsing logic as much as possible.